src/proxyManagerConfiguration.js

```diff
@@ -51,14 +65,17 @@ const proxyManagerConfiguration = {
           viewUp: [0, 0, 1], // Z+ (S)
           useParallelRendering: false,
         },
+        props: {
+          presetToInteractor3D: interactorStyle3D,
+          presetToInteractor2D: interactorStyle2D,
+        },
```

Comment:
Just to be sure. Do you have a `setPresetToInteractor2/3D(...)` method? If so, then you are good.

Reply:
Yes, @agirault set these, and I found that it works in practice :-). They are defined in vtk.js here: 

  https://github.com/Kitware/vtk-js/blob/41b0da325781d0a3bbe082a22927ec9fbea53bef/Sources/Proxy/Core/ViewProxy/index.js#L77-L118

In general, it would be nice to have documentation for the proxy manager configuration.